example/classifier/odp_classifier.c

```diff
@@ -187,6 +187,9 @@ static inline void print_cls_statistics(appl_args_t *args)
 		infinite = 1;
 
 	for (; timeout > 0 || infinite; timeout--) {
+		sleep(1);
+		printf("\r");
+		fflush(stdout);
```

Comment:
The flushing could be moved to just before the `shutdown_sig` check.

Reply:
Indeed. Will move it.